core/models/asset.py

```diff
@@ -552,9 +552,36 @@ class AssetIncidentReport(models.Model):
     witnesses = models.TextField(null=True, blank=True)
     police_abstract_obtained = models.CharField(max_length=255)
     submitted_by = models.ForeignKey('User', null=True, on_delete=models.PROTECT)
+    state = models.CharField(max_length=50)
```

Comment:
i think we can do without this field since the `StateTransition` model has a foreign key pointing to this table